README.md

```diff
@@ -317,43 +317,23 @@ that wish to use it as a module. A bare-minimum flake.nix would be as follows:
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-    home-manager.url = "github:rycee/home-manager";
+    home-manager.url = "github:nix-community/home-manager";
   };
 
-  outputs = inputs: {
+  outputs = { home-manager, nixpkgs, ... }: {
     nixosConfigurations = {
-      hostname = let
+      container = nixpkgs.lib.nixosSystem {
```

Comment:
```suggestion
      hostname = nixpkgs.lib.nixosSystem {
```

I think this should still be `hostname`, because it makes `nixos-rebuild --flake .` easier -- by default, it uses the hostname if nothing else is specified.

Reply:
I was checking it on a nixos-container, so yeah)